chalice/utils.py

```diff
@@ -101,6 +116,84 @@ def set_file_contents(self, filename, contents, binary=True):
         with open(filename, mode) as f:
             f.write(contents)
 
+    def unpack_zipfile(self, zipfile_path, unpack_dir):
+        # type: (str, str) -> None
+        with zipfile.ZipFile(zipfile_path, 'r') as z:
+            z.extractall(unpack_dir)
+
+    @contextlib.contextmanager
+    def zipfile_context(self, filename, mode='r',
```

Comment:
Why do we need to wrap the `zipfile` with its own context manager?  This is already a capability of `zipfile.ZipFile` so it seems unnecessary.